cluster.go

```diff
@@ -43,6 +48,7 @@ func NewCluster(dialer gocql.HostDialer, username, password string) *gocql.Clust
 	cluster.Authenticator = &gocql.PasswordAuthenticator{
 		Username: username,
 		Password: password,
+		AllowedAuthenticators: []string{apacheAuthenticator, dseAuthenticator, astraAuthenticator},
```

Comment:
justfyi [this does remove some of the default authenticators](https://github.com/gocql/gocql/blob/34fdeebefcbf183ed7f916f931aa0586fdaa1b40/conn.go#L27-L38) but the ones that are being removed with this are not used in Astra so it's fine